source/element-kinds.md

```diff
@@ -313,8 +313,7 @@ Element parameter groups associated with this element kind are:
 ## Marker Element
 
 Zero length element to mark a particular position.
-
-Under Construction...
+The main purpose of this element is to name a position in the beamline.
 
 Element parameter groups associated with this element kind are:
```

Comment:
Do we really want these parameter groups on `Marker`?

Reply:
The idea is that a `Marker` element has a unit transport map (that is, does not affect the particle phase space coordinates). So it is not "generic" in the sense that it could represent a zero length kicker. However, for example, using markers for BPMs would be OK.

I don't know what is meant by "Do we really want these parameter groups on Marker?"

Reply:
Yes I agree. I would take out `ReferenceChangeP` as a possible group for `Marker` elements.